legacy/oled.c

```
@@ -59,7 +59,10 @@
 
 static uint8_t _oledbuffer[OLED_BUFSIZE];
 static uint8_t _oledbuffer_bak[OLED_BUFSIZE];
+static uint8_t _oledbuffer_overlay[OLED_BUFSIZE];
```

Clang (14.0.6)

[warning] declaration uses identifier '_oledbuffer_overlay', which is reserved in the global namespace

(bugprone-reserved-identifier)